packages/cli/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Zowe CLI package will be documented in this file.
 
 ## Recent Changes
 
+- Enhancement: When copying PDSs that have identical member names, the user is now prompted to confirm before the operation occurs in case of data loss. [#2349] (https://github.com/zowe/zowe-cli/issues/2349)
```

Comment:
- Let's write out the PDSs: partitioned data sets
- might be clearer to say: " ... is now prompted to confirm the operation in case of potential data loss."